src/base/iana/exterr.rs

```diff
@@ -149,6 +149,9 @@ int_enum! {
     /// is otherwise configured to support. Examples of this include
     /// its most recent zone being too old or having expired.
     (INVALID_DATA => 24, b"Invalid Data")
+
+    /// The requested resource record should not appear in query
```

Comment:
tiniest nit: comment should end with a `.`

Reply:
Haha no worries, I've added it